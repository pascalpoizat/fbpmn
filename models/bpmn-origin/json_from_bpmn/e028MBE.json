{
    "isInterrupt": {
        "BoundaryEvent_1fak9ar": false,
        "BoundaryEvent_0k5p91i": true,
        "BoundaryEvent_1q0fgiw": true
    },
    "catN": {
        "BoundaryEvent_1fak9ar": "MessageBoundaryEvent",
        "SubProcess_0joqwjm": "SubProcess",
        "Task_06osngf": "AbstractTask",
        "EndEvent_112jhwq": "NoneEndEvent",
        "StartEvent_1": "NoneStartEvent",
        "ExclusiveGateway_0hzcbkt": "XorGateway",
        "EndEvent_1hytbgh": "NoneEndEvent",
        "Task_1i35ppr": "AbstractTask",
        "StartEvent_0ef0xw9": "NoneStartEvent",
        "EndEvent_1gf9mha": "NoneEndEvent",
        "Task_0z3cxl5": "SendTask",
        "BoundaryEvent_0k5p91i": "MessageBoundaryEvent",
        "BoundaryEvent_1q0fgiw": "MessageBoundaryEvent",
        "P_": "Process",
        "ExclusiveGateway_0pvsvob": "XorGateway",
        "ExclusiveGateway_1vvyptg": "XorGateway",
        "EndEvent_1whel54": "NoneEndEvent",
        "SubProcess_07e2e99": "SubProcess",
        "ExclusiveGateway_1hukmk5": "XorGateway",
        "StartEvent_1jxrjjb": "NoneStartEvent",
        "EndEvent_0ovxd43": "MessageEndEvent",
        "Task_1kso3jk": "AbstractTask",
        "StartEvent_03bqw0j": "NoneStartEvent",
        "Q_": "Process",
        "Task_0ouyduq": "SendTask"
    },
    "catE": {
        "SequenceFlow_17254s0": "NormalSequenceFlow",
        "SequenceFlow_0y8q6ot": "NormalSequenceFlow",
        "SequenceFlow_1sp2uu5": "NormalSequenceFlow",
        "SequenceFlow_1hr9yu6": "NormalSequenceFlow",
        "SequenceFlow_1rnwbjb": "NormalSequenceFlow",
        "SequenceFlow_0fcn4he": "NormalSequenceFlow",
        "SequenceFlow_03zwnxj": "NormalSequenceFlow",
        "SequenceFlow_00h8rbi": "DefaultSequenceFlow",
        "SequenceFlow_18dl2c1": "ConditionalSequenceFlow",
        "MessageFlow_1fv5g0n": "MessageFlow",
        "SequenceFlow_1xvizyy": "NormalSequenceFlow",
        "SequenceFlow_06cnlpk": "DefaultSequenceFlow",
        "SequenceFlow_109652j": "NormalSequenceFlow",
        "SequenceFlow_08qsdvs": "NormalSequenceFlow",
        "SequenceFlow_1hqd2qa": "NormalSequenceFlow",
        "MessageFlow_01dn8b3": "MessageFlow",
        "MessageFlow_01rg2pg": "MessageFlow",
        "SequenceFlow_1ag4q25": "NormalSequenceFlow",
        "SequenceFlow_0axwuh9": "ConditionalSequenceFlow",
        "SequenceFlow_1es2p0l": "NormalSequenceFlow",
        "SequenceFlow_0e70fm8": "NormalSequenceFlow",
        "SequenceFlow_16cdiue": "NormalSequenceFlow",
        "SequenceFlow_1i24zsv": "NormalSequenceFlow"
    },
    "containN": {
        "SubProcess_0joqwjm": [
            "ExclusiveGateway_0pvsvob",
            "ExclusiveGateway_1vvyptg",
            "Task_0ouyduq",
            "Task_0z3cxl5",
            "StartEvent_0ef0xw9",
            "EndEvent_1whel54"
        ],
        "P_": [
            "SubProcess_07e2e99",
            "StartEvent_1",
            "EndEvent_1hytbgh",
            "Task_06osngf",
            "EndEvent_112jhwq",
            "BoundaryEvent_1fak9ar",
            "BoundaryEvent_1q0fgiw"
        ],
        "SubProcess_07e2e99": [
            "StartEvent_03bqw0j",
            "Task_1i35ppr",
            "Task_1kso3jk",
            "ExclusiveGateway_0hzcbkt",
            "ExclusiveGateway_1hukmk5",
            "EndEvent_0ovxd43"
        ],
        "Q_": [
            "SubProcess_0joqwjm",
            "StartEvent_1jxrjjb",
            "EndEvent_1gf9mha",
            "BoundaryEvent_0k5p91i"
        ]
    },
    "messageE": {
        "MessageFlow_1fv5g0n": "msg2",
        "MessageFlow_01dn8b3": "msg1",
        "MessageFlow_01rg2pg": "msg3"
    },
    "timeInformation": {},
    "targetE": {
        "SequenceFlow_17254s0": "ExclusiveGateway_1vvyptg",
        "SequenceFlow_0y8q6ot": "EndEvent_112jhwq",
        "SequenceFlow_1sp2uu5": "EndEvent_1hytbgh",
        "SequenceFlow_1hr9yu6": "EndEvent_1whel54",
        "SequenceFlow_1rnwbjb": "SubProcess_07e2e99",
        "SequenceFlow_0fcn4he": "ExclusiveGateway_0hzcbkt",
        "SequenceFlow_03zwnxj": "ExclusiveGateway_0hzcbkt",
        "SequenceFlow_00h8rbi": "Task_1kso3jk",
        "SequenceFlow_18dl2c1": "Task_1i35ppr",
        "MessageFlow_1fv5g0n": "BoundaryEvent_1fak9ar",
        "SequenceFlow_1xvizyy": "EndEvent_1gf9mha",
        "SequenceFlow_06cnlpk": "Task_0z3cxl5",
        "SequenceFlow_109652j": "Task_06osngf",
        "SequenceFlow_08qsdvs": "ExclusiveGateway_1vvyptg",
        "SequenceFlow_1hqd2qa": "EndEvent_1gf9mha",
        "MessageFlow_01dn8b3": "BoundaryEvent_1q0fgiw",
        "MessageFlow_01rg2pg": "BoundaryEvent_0k5p91i",
        "SequenceFlow_1ag4q25": "EndEvent_1hytbgh",
        "SequenceFlow_0axwuh9": "Task_0ouyduq",
        "SequenceFlow_1es2p0l": "SubProcess_0joqwjm",
        "SequenceFlow_0e70fm8": "EndEvent_0ovxd43",
        "SequenceFlow_16cdiue": "ExclusiveGateway_1hukmk5",
        "SequenceFlow_1i24zsv": "ExclusiveGateway_0pvsvob"
    },
    "name": "e028MBE",
    "attached": {
        "BoundaryEvent_1fak9ar": "SubProcess_07e2e99",
        "BoundaryEvent_0k5p91i": "SubProcess_0joqwjm",
        "BoundaryEvent_1q0fgiw": "SubProcess_07e2e99"
    },
    "containE": {
        "SubProcess_0joqwjm": [
            "SequenceFlow_06cnlpk",
            "SequenceFlow_0axwuh9",
            "SequenceFlow_17254s0",
            "SequenceFlow_08qsdvs",
            "SequenceFlow_1i24zsv",
            "SequenceFlow_1hr9yu6"
        ],
        "P_": [
            "SequenceFlow_1rnwbjb",
            "SequenceFlow_1ag4q25",
            "SequenceFlow_1sp2uu5",
            "SequenceFlow_0y8q6ot",
            "SequenceFlow_109652j"
        ],
        "SubProcess_07e2e99": [
            "SequenceFlow_16cdiue",
            "SequenceFlow_18dl2c1",
            "SequenceFlow_00h8rbi",
            "SequenceFlow_03zwnxj",
            "SequenceFlow_0fcn4he",
            "SequenceFlow_0e70fm8"
        ],
        "Q_": [
            "SequenceFlow_1hqd2qa",
            "SequenceFlow_1es2p0l",
            "SequenceFlow_1xvizyy"
        ]
    },
    "edges": [
        "MessageFlow_01dn8b3",
        "MessageFlow_1fv5g0n",
        "MessageFlow_01rg2pg",
        "SequenceFlow_1rnwbjb",
        "SequenceFlow_1ag4q25",
        "SequenceFlow_1sp2uu5",
        "SequenceFlow_0y8q6ot",
        "SequenceFlow_109652j",
        "SequenceFlow_16cdiue",
        "SequenceFlow_18dl2c1",
        "SequenceFlow_00h8rbi",
        "SequenceFlow_03zwnxj",
        "SequenceFlow_0fcn4he",
        "SequenceFlow_0e70fm8",
        "SequenceFlow_1hqd2qa",
        "SequenceFlow_1es2p0l",
        "SequenceFlow_1xvizyy",
        "SequenceFlow_06cnlpk",
        "SequenceFlow_0axwuh9",
        "SequenceFlow_17254s0",
        "SequenceFlow_08qsdvs",
        "SequenceFlow_1i24zsv",
        "SequenceFlow_1hr9yu6"
    ],
    "messages": [
        "msg1",
        "msg2",
        "msg3"
    ],
    "sourceE": {
        "SequenceFlow_17254s0": "Task_0z3cxl5",
        "SequenceFlow_0y8q6ot": "Task_06osngf",
        "SequenceFlow_1sp2uu5": "BoundaryEvent_1q0fgiw",
        "SequenceFlow_1hr9yu6": "ExclusiveGateway_1vvyptg",
        "SequenceFlow_1rnwbjb": "StartEvent_1",
        "SequenceFlow_0fcn4he": "Task_1kso3jk",
        "SequenceFlow_03zwnxj": "Task_1i35ppr",
        "SequenceFlow_00h8rbi": "ExclusiveGateway_1hukmk5",
        "SequenceFlow_18dl2c1": "ExclusiveGateway_1hukmk5",
        "MessageFlow_1fv5g0n": "Task_0z3cxl5",
        "SequenceFlow_1xvizyy": "SubProcess_0joqwjm",
        "SequenceFlow_06cnlpk": "ExclusiveGateway_0pvsvob",
        "SequenceFlow_109652j": "BoundaryEvent_1fak9ar",
        "SequenceFlow_08qsdvs": "Task_0ouyduq",
        "SequenceFlow_1hqd2qa": "BoundaryEvent_0k5p91i",
        "MessageFlow_01dn8b3": "Task_0ouyduq",
        "MessageFlow_01rg2pg": "EndEvent_0ovxd43",
        "SequenceFlow_1ag4q25": "SubProcess_07e2e99",
        "SequenceFlow_0axwuh9": "ExclusiveGateway_0pvsvob",
        "SequenceFlow_1es2p0l": "StartEvent_1jxrjjb",
        "SequenceFlow_0e70fm8": "ExclusiveGateway_0hzcbkt",
        "SequenceFlow_16cdiue": "StartEvent_03bqw0j",
        "SequenceFlow_1i24zsv": "StartEvent_0ef0xw9"
    },
    "nodes": [
        "P_",
        "Q_",
        "SubProcess_07e2e99",
        "StartEvent_1",
        "EndEvent_1hytbgh",
        "Task_06osngf",
        "EndEvent_112jhwq",
        "BoundaryEvent_1fak9ar",
        "BoundaryEvent_1q0fgiw",
        "StartEvent_03bqw0j",
        "Task_1i35ppr",
        "Task_1kso3jk",
        "ExclusiveGateway_0hzcbkt",
        "ExclusiveGateway_1hukmk5",
        "EndEvent_0ovxd43",
        "SubProcess_0joqwjm",
        "StartEvent_1jxrjjb",
        "EndEvent_1gf9mha",
        "BoundaryEvent_0k5p91i",
        "ExclusiveGateway_0pvsvob",
        "ExclusiveGateway_1vvyptg",
        "Task_0ouyduq",
        "Task_0z3cxl5",
        "StartEvent_0ef0xw9",
        "EndEvent_1whel54"
    ],
    "nameN": {
        "Task_06osngf": "C",
        "Task_1i35ppr": "A",
        "Task_0z3cxl5": "Send Message 2",
        "Task_1kso3jk": "B",
        "Task_0ouyduq": "Send Message 1"
    }
}