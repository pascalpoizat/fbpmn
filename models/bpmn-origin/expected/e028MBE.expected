Working in /tmp/e028MBE.T1f31 with 1 worker(s)
[92mtransformation done[0m
<<"Processes=", 2, "Nodes=", 25, "Gateway=", 4, "SF=", 20, "MF=", 3>>
---------- Network01Bag ----------
[X] Prop01Type
     states=543 trans=1334 depth=25
[X] Prop02Safe
     states=543 trans=1334 depth=25
[ ] Prop03Sound
     states=543 trans=1334 depth=
[X] Prop04MsgSound
     states=543 trans=1334 depth=25
---------- Network02FifoPair ----------
[X] Prop01Type
     states=544 trans=1334 depth=25
[X] Prop02Safe
     states=544 trans=1334 depth=25
[ ] Prop03Sound
     states=544 trans=1334 depth=
[X] Prop04MsgSound
     states=544 trans=1334 depth=25
---------- Network03Causal ----------
[X] Prop01Type
     states=604 trans=1491 depth=25
[X] Prop02Safe
     states=604 trans=1491 depth=25
[ ] Prop03Sound
     states=604 trans=1491 depth=
[X] Prop04MsgSound
     states=604 trans=1491 depth=25
---------- Network04Inbox ----------
[X] Prop01Type
     states=544 trans=1334 depth=25
[X] Prop02Safe
     states=544 trans=1334 depth=25
[ ] Prop03Sound
     states=544 trans=1334 depth=
[X] Prop04MsgSound
     states=544 trans=1334 depth=25
---------- Network05Outbox ----------
[X] Prop01Type
     states=544 trans=1334 depth=25
[X] Prop02Safe
     states=544 trans=1334 depth=25
[ ] Prop03Sound
     states=544 trans=1334 depth=
[X] Prop04MsgSound
     states=544 trans=1334 depth=25
---------- Network06Fifo ----------
[X] Prop01Type
     states=573 trans=1378 depth=25
[X] Prop02Safe
     states=573 trans=1378 depth=25
[ ] Prop03Sound
     states=573 trans=1378 depth=
[X] Prop04MsgSound
     states=573 trans=1378 depth=25
---------- Network07RSC ----------
[X] Prop01Type
     states=501 trans=1241 depth=25
[X] Prop02Safe
     states=501 trans=1241 depth=25
[ ] Prop03Sound
     states=501 trans=1241 depth=
[X] Prop04MsgSound
     states=501 trans=1241 depth=25
done.
